build/Microsoft.DotNet.Cli.GitCommitInfo.targets

```diff
@@ -0,0 +1,44 @@
+<Project ToolsVersion="15.0">
+  <Target Name="WriteGitCommitInfoProps"
+          Condition=" !Exists('$(GitCommitInfoProps)') "
```

Comment:
This condition will prevent refreshing the CommitInfoProps, won't it? I think we need to remove it.